sgdk/Makefile

```diff
@@ -24,21 +23,17 @@ patch: $(SGDKDIR)
 prep: $(SGDKDIR)
 	cd $(SGDKDIR) && patch -u -p1 -l < ../files/sgdk_$(SGDKVER).diff
 
+.PHONY: $(SGDKDIR)
 SGDK: $(SGDKDIR)
 $(SGDKDIR):
-	git clone -b '$(SGDKVER)' --single-branch --depth 1 https://github.com/Stephane-D/SGDK.git
+	git submodule update --init
```

Comment:
We'll want this to represent the SGDK version we want on line 7.  (We may not always want the very latest, and also that SGDKVER is used to patch our local SGDK dir for specific tweaks need to run on *nix)

Maybe `git submodule -b tags/$(SGDKVER) ......` before the module update is what we need.